tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -68,10 +68,6 @@ acl/null_route/test_null_route_helper.py:
       - "'m1' in topo_type"
 
 acl/test_acl.py:
-  skip:
-    reason: "Skip acl for isolated-v6 topology"
-    conditions:
-      - "'isolated-v6' in topo_name and https://github.com/sonic-net/sonic-mgmt/issues/18077"
```

Comment:
hi @wrideout-arista , do you mind to help limit this to Arista / broadcom?

Reply:
checked with David, with the fix, the ACL test pass rate will be increased from 20% to 75% (not 100% with skips), hence let's keep the skip for now and we can flip is in later release quals.